packages/trpc/server/routers/viewer/teams/create.handler.ts

```diff
@@ -16,11 +16,20 @@ type CreateOptions = {
 };
 
 export const createHandler = async ({ ctx, input }: CreateOptions) => {
+  const { user } = ctx;
   const { slug, name, logo } = input;
+  const isOrgChildTeam = !!user.organizationId;
+
+  // For orgs we want to create teams under the org
+  if (user.organizationId && !user.organization.isOrgAdmin) {
+    throw new TRPCError({ code: "FORBIDDEN", message: "org_admins_can_create_new_teams" });
+  }
```

Comment:
Rather than having a long query to check if the user can create a team, just use the new context added.